web/client/localConfig.json

```diff
@@ -386,7 +386,19 @@
                 "declineUrl" : "http://www.google.com"
                 }
             },
-            "OmniBar", "Login", "Save", "SaveAs", "BurgerMenu", "Expander", "Undo", "Redo", "FullScreen", "GlobeViewSwitcher", "SearchServicesConfig", "WidgetsBuilder", "Widgets", "WidgetsTray",
+            "OmniBar", "Login", "Save", "SaveAs", "BurgerMenu",
+            "Expander", "Undo", "Redo", "FullScreen", "GlobeViewSwitcher", "SearchServicesConfig", {
+                "name": "WidgetsBuilder",
+                "cfg": {
+                    "shortenChartLabelThreshold": 1000
```

Comment:
Hi, please do not push changes here, because it is better to not add too much stuff in the configuration.
For our purpose it is enough to use a default value at the plugin level